src/chainparams.cpp

```diff
@@ -162,10 +162,10 @@ class CMainParams : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            // Data from rpc: getchaintxstats 16384 3a9d0a6b19f4c20959caef1bad1a7aa18e492c049327ccaf3b3fcf7c3815f4e0
-            1648016191, // nTime
-            4991331,  // nTxCount
-            0.01460440502969592 // dTxRate
+            // Data from rpc: getchaintxstats 16384 8d45b551b45904d28ba69dc7112ff7793e5f229d9f1eb10bd105afbca9c4b783
```

Comment:
This comment should be updated for block 2046000
// Data from rpc: getchaintxstats 2046000